processor/probabilisticsamplerprocessor/testdata/config.yaml

```diff
@@ -3,8 +3,8 @@ receivers:
 
 processors:
   probabilistic-sampler:
```

Comment:
I think we also need to do the same replacement in processor names, i.e. `probabilistic_sampler`